src/sentry/models/dashboard_widget.py

```diff
@@ -34,6 +34,16 @@ def get_id_for_type_name(cls, type_name):
                 return id
 
 
+class DashboardWidgetTypes(TypesClass):
+    DISCOVER = 0
+    ISSUE = 1
+    TYPES = [
+        (DISCOVER, "discover"),
+        (ISSUE, "issue"),
```

Comment:
In the future could this be extended for metrics queries? cc @matejminar @priscilawebdev